docs/examples/datamover_advanced_voloptions.md

```diff
@@ -360,6 +360,10 @@ spec:
             storageClassName: ocs-storagecluster-cephfs-shallow
           destinationVolumeOptions:
             moverSecurityContext: true
+            accessMode: ReadOnlyMany
```

Comment:
Does this work on ceph? It doesn't for gp2.

Reply:
https://github.com/ceph/ceph-csi/blob/devel/docs/cephfs-snapshot-backed-volumes.md